### DISCUSSION TITLE
Bug: theming issue on showcase page

### DISCUSSION THREAD

**Discussion by servusdei2018** [community/community#123316] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The showcases page ["Great for new contributors"](https://github.com/showcases/great-for-new-contributors) looks fine on light mode themes but on all dark mode themes it displays a very low-contrast banner in light-mode:

![Screenshot 2024-05-10 at 14-30-49 Great for new contributors](https://github.com/community/community/assets/37554478/f9783a05-231a-4294-b81a-479580eb3921)

**Comment by ebndev** [comment]
Hey @servusdei2018 - thanks for flagging this!
